packages/flutter_tools/test/integration.shard/test_data/multidex_project.dart

```diff
@@ -179,7 +179,7 @@ class MultidexProject extends Project {
       defaultConfig {
           // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
           applicationId "com.example.multidextest2"
-          minSdkVersion 16
+          minSdkVersion 19
```

Comment:
@blasten From test results I see that `cloud_firestore`, used in this test project, requires `minSdkVersion 19`. I am not sure why this test fails now instead of failing in past builds as well.

Reply:
this could happen if a dependency is resolved to a newer version that has minSdkVersion:19.

Reply:
Yes, but I still wanted to highlight a few of the issues/fixes I think you should be aware of. With the lockfiles, there's a lot of modified files and I would not want such things to go unnoticed.